app/routes/_gcn.docs.circulars.markdown.mdx

```diff
@@ -0,0 +1,307 @@
+
+<table class="usa-table">
+  <thead>
+    <tr>
+      <th>Example</th>
+      <th>Output</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>This is \*\*Bold\*\*</td>
+    <td>This is **Bold**</td>
+  </tr>
+  <tr>
+    <td>This is \*Italic\*</td>
+    <td>This is *Italic*</td>
+  </tr>
+  <tr>
+    <td>\*\*This is \*Italic\* in Bold\*\*</td>
+    <td>**This is *Italic* in Bold**</td>
+  </tr>
+  <tr>
+    <td>This is \*\*\*Bold and Italic\*\*\*</td>
+    <td>This is ***Bold and Italic***</td>
+  </tr>
+  <tr>
+    <td>
+      This is a \[Link\]\(https://<span></span>gcn.<span></span>nasa.gov\)
+    </td>
+    <td>This is a [Link](https://gcn.nasa.gov)</td>
+  </tr>
+  <tr>
+    <td>
+      A Link to &lt;https://<span></span>gcn.<span></span>nasa.gov&gt;
+    </td>
+    <td>A Link to [https://gcn.nasa.gov](https://gcn.nasa.gov)</td>
+  </tr>
+</table>
+
+### Headers
+
+There are six levels of headers in markdown. The number of `#` symbols at the beginning of a line determines the level of the header, with the first level being the largest and the sixth level being the smallest.
+
+<table class="usa-table">
+  <thead>
+    <tr>
+      <th>Example</th>
+      <th>Output</th>
+    </tr>
+  </thead>
+  <tr>
+    <td># Header 1</td>
+    <td>
+      <h1>Header 1</h1>
+    </td>
+  </tr>
+  <tr>
+    <td>## Header 2</td>
+    <td>
+      <h2>Header 2</h2>
+    </td>
+  </tr>
+  <tr>
+    <td>### Header 3</td>
+    <td>
+      <h3>Header 3</h3>
+    </td>
+  </tr>
+  <tr>
+    <td>#### Header 4</td>
+    <td>
+      <h4>Header 4</h4>
+    </td>
+  </tr>
+  <tr>
+    <td>##### Header 5</td>
+    <td>
+      <h5>Header 5</h5>
+    </td>
+  </tr>
+  <tr>
+    <td>###### Header 6</td>
+    <td>
+      <h6>Header 6</h6>
+    </td>
+  </tr>
+</table>
+
+Header levels 1 and 2 can be created using an alternate method; the first and second levels of header is created by following the header text with a line of `=` or `-` symbols, respectively.
+
+<table class="usa-table">
+  <thead>
+    <tr>
+      <th>Example</th>
+      <th>Output</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>
+      Header 1<br></br>========
+    </td>
+    <td>
+      <h1>Header 1</h1>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Header 2<br></br>--------
+    </td>
+    <td>
+      <h2>Header 2</h2>
+    </td>
+  </tr>
+</table>
+
+### Lists
+
+Markdown supports both ordered and unordered lists. Unordered lists can be created using the `*`, `-`, or `+` symbols, while ordered lists can be created using numbers followed by a period.
+
+<table class="usa-table">
+  <thead>
+    <tr>
+      <th>Example</th>
+      <th>Output</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>
+      \-/\*/+ Item 1<br></br>\-/\*/+ Item 2<br></br>\-/\*/+ Item 3
```

Comment:
Please use block code formatting for block syntax examples.

```suggestion
      ```
      - Item 1
      * Item 2
      + Item 3
      ```
```